scipy/stats/_distn_infrastructure.py

```diff
@@ -2894,8 +2939,6 @@ def __init__(self, a=0, b=inf, name=None, badvalue=None,
         # now that self.numargs is defined, we can adjust nin
         self._cdfvec.nin = self.numargs + 1
 
-        self._construct_docstrings(name, longname, extradoc)
```

Comment:
why removing it?

Reply:
It's not actually removed, it's the diff that looks funny. If you look at the entire file `_construct_docstrings` is still there.